js/test/test.js

```diff
@@ -107,12 +107,40 @@ if (settings && settings.skip) {
     process.exit ();
 }
 
-//-----------------------------------------------------------------------------
+
+// ### common language specific methods ###
+
+async function runTesterMethod(exchange, methodName, ... args) {
+    return await tests[methodName](exchange, ... args);
+}
+
+function testMethodAvailableForCurrentLang(methodName) {
+    return tests[methodName] !== undefined;
+}
+
+function findValueIndexInArray (arr, value) {
+    return arr.indexOf (value);
+}
+
+function exceptionHint (exc) {
+    return '[' + exc.constructor.name + '] ' + exc.message.slice (0, 200);
+}
+// ### end of language specific common methods ###
+
```

Comment:
these above added methods are currently used only in JS (`test.js`) because it is not yet transpiled to other langs (`test.php, test.py`) so, after it will be transpiled, these methods will be used there too